regen/mg_vtable.pl

```diff
@@ -212,6 +212,9 @@ BEGIN
                    vtable => 'debugvar' },
      lvref => { char => '\\', vtable => 'lvref',
                   desc => "Lvalue reference constructor" },
+     destruct => { char => "X", vtable => 'destruct',
+                        desc => "destruct callback" },
```

Comment:
This should probably also set «value_magic => 1», otherwise localizing such a value will cause the free handler to be run twice

Reply:
Done.